Assets/HoloToolkit/Input/Scripts/GazeStabilizer.cs

```diff
@@ -136,18 +136,18 @@ private void UpdateInstability(out float positionInstability, out float directio
                 return;
             }
 
-            mostRecentSample = stabilitySamples.ElementAt(stabilitySamples.Count - 1);
+            mostRecentSample = stabilitySamples.Last.Value;
```

Comment:
.Last is good I guess as its faster than .Last() which can be O(n) for some collections.